packages/clients/test/aptos/aptos.integration.test.ts

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright 2023-2024 Shinami Corp.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { describe, expect, it } from "@jest/globals";
+import { createAptos } from "./integration.env";
+import { SingleKeyAccount, SigningSchemeInput } from "@aptos-labs/ts-sdk";
+
+const aptos = createAptos();
+
+describe("Shinami Aptos client", () => {
+  it("should query via REST API", async () => {
+    const state = await aptos.getLedgerInfo();
+    expect(state).toMatchObject({
+      block_height: expect.stringMatching(/[0-9]+/),
+      chain_id: /[0-9]+/,
+      epoch: expect.stringMatching(/[0-9]+/),
+      git_hash: expect.stringMatching(/[0-9a-f]{5,40}/),
+      ledger_timestamp: expect.stringMatching(/[0-9]+/),
+      node_role: expect.stringMatching("full_node"),
+      oldest_block_height: expect.stringMatching(/[0-9]+/),
+      oldest_ledger_version: expect.stringMatching(/[0-9]+/),
+    });
+  });
+
+  it("should query GraphQL successfully", async () => {
+    const ledgerInfo = await aptos.queryIndexer({
+      query: {
+        query: `
+                query IntegrationTestQuery {
+                  ledger_infos {
+                    chain_id
+                  }
+                }
+              `,
+      },
+    });
+    expect(ledgerInfo).toMatchObject({
+      ledger_infos: [
+        {
+          chain_id: /[0-2]+/,
```

Comment:
Which chain ids should be matched here? I think this regex can currently match chain id `202` for example

Reply:
Good catch, I am looking for integer matches of 0, 1, and 2, representing mainnet, testnet and devnet respectively. I can change to have anchors on the beginning and end to confirm the match

Reply:
I believe 1 and 2 are mainnet and testnet respectively: https://aptos.dev/en/network/nodes/networks
I don't think chain id 0 is allowed

Reply:
https://github.com/aptos-labs/aptos-core/blob/3e7bf94b34f5dc8d9bcd6615437f9c51a9af3155/types/src/chain_id.rs#L183